bin/next-dev

```diff
@@ -60,7 +61,9 @@ const srv = new Server({ dir, dev: true })
 srv.start(argv.port, argv.hostname)
 .then(async () => {
   if (!process.env.NOW) {
-    console.log(`> Ready on http://${argv.hostname ? argv.hostname : 'localhost'}:${argv.port}`)
+    const url = `http://${argv.hostname ? argv.hostname : 'localhost'}:${argv.port}`
+    copySync(url)
```

Comment:
Lets do it the same way as micro-dev https://github.com/zeit/micro-dev/blob/3d264d50dc0d30ea88a9563287e020a112e67365/lib/listening.js#L13-L20

Reply:
@timneutkens Is it okay if I put this `copyToClipboard` in a new `bin/utils.js`?
If I export it from `lib/utils.js`, I get this error when running `next`:

```
 ERROR  Failed to compile with 6 errors                                                                                                                                               17:38:21

These dependencies were not found:

* child_process in D:/Projects/next.js/node_modules/cross-spawn/index.js, D:/Projects/next.js/node_modules/execa/index.js
* fs in D:/Projects/next.js/node_modules/cross-spawn/lib/util/readShebang.js, D:/Projects/next.js/node_modules/isexe/index.js and 2 others

To install them, you can run: npm install --save child_process fs
```